sample-compose/app/src/main/java/co/nimblehq/sample/compose/model/UiModel.kt

```diff
@@ -1,10 +1,13 @@
 package co.nimblehq.sample.compose.model
 
+import android.os.Parcelable
 import co.nimblehq.sample.compose.domain.model.Model
+import kotlinx.parcelize.Parcelize
 
+@Parcelize
 data class UiModel(
     val id: Int
-)
+) : Parcelable
 
 private fun Model.toUiModel() = UiModel(id = id ?: -1)
```

Comment:
The default `-1` could lead the text shows the `-1` string if I'm not wrong. 🤔  Why don't we declare the `UiModel.id` to be String and set the empty string as the default since we only use it to display?

Reply:
Good point @lydiasama 👍 I agree but I'm not sure should we follow the `template` or not for this 🤔

Reply:
@Wadeewee I don't think we need to keep aligned with the template in this case. Because in the template, we don't use this value as a sample. 🤔 Also, some decision depends on the project independently. ([See the discussion here](https://github.com/nimblehq/android-templates/pull/274#discussion_r948629123))